File: Unidad 4/p6_u4.cs
using System;

class Program
{
    static void Main()
    {
        int[] numeros = { 4, 2, 9, 1, 8, 0, 3, 6 };

        int[] arregloOrdenado = OrdenarArreglo(numeros);

        Console.WriteLine("Arreglo Ordenado:");
        foreach (int num in arregloOrdenado)
        {
            Console.Write(num + " ");
        }
    }

    static int[] OrdenarArreglo(int[] arreglo)
    {
        int[] arregloOrdenado = new int[arreglo.Length];
        Array.Copy(arreglo, arregloOrdenado, arreglo.Length);

        for (int i = 0; i < arregloOrdenado.Length - 1; i++)
        {
            for (int j = 0; j < arregloOrdenado.Length - 1 - i; j++)
            {
                if (arregloOrdenado[j] > arregloOrdenado[j + 1])
                {
                    int temp = arregloOrdenado[j];
                    arregloOrdenado[j] = arregloOrdenado[j + 1];
                    arregloOrdenado[j + 1] = temp;
                }
            }
        }

        return arregloOrdenado;
    }
}

File: Unidad 4/P6_u4.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.p6_u4;
import java.util.Arrays;

 
/**
 *
 * @author richy
 */
public class P6_u4 {

    public static void main(String[] args) {
            int[] numeros = { 4, 2, 9, 1, 8, 0, 3, 6 };

        Arrays.sort(numeros);

        System.out.println("Arreglo Ordenado:");
        for (int num : numeros) {
            System.out.print(num + " ");
        }
    }
}